lib/clAdapters/google-mail/index.js

```diff
@@ -274,6 +274,16 @@ var hasLabel = function(message, labelValue) {
   return message.labelIds && message.labelIds.length && (message.labelIds.indexOf(labelValue) >= 0);
 };
 
+var getFirstScalarPart = function(partToCheck) {
+  var returnObject = partToCheck;
+  _.forEach(partToCheck.headers, function(header) {
```

Comment:
I don't know much about email; can you have more than one Content-Type headers? If not, it might be faster to use a _.find, especially since this is recursive.